bundles/org.eclipse.passage.lic.jetty/src/org/eclipse/passage/lic/internal/jetty/interaction/ServerHandles.java

```diff
@@ -62,7 +62,8 @@ public void restart() {
 
 	public void state() {
 		try {
-			System.out.println(server.state());
+			String where = port.get().map(i -> i.toString()).orElse("-"); //$NON-NLS-1$
+			System.out.println(server.state() + " on port " + where); //$NON-NLS-1$
```

Comment:
System.out?

Reply:
Yes. This is console commant, it speaks to a user through osgi console.